public/docs/ts/latest/guide/user-input.jade

```diff
@@ -69,15 +72,15 @@ figure.image-display
 <a id="keyup1"></a>
 .l-sub-section
   :marked
-    We cast the `$event` as an `any` type, which means we've abandoned strong typing
-    to simplify our code. We generally prefer the strong typing that TypeScript affords.
-    We can rewrite the method, casting to HTML DOM objects like this.
+    You cast the `$event` as an `any` type, which means you have abandoned strong typing
+    to simplify your code.
+    You can rewrite the method, casting to HTML DOM objects as follows:
   +makeExample('user-input/ts/app/keyup.components.ts', 'key-up-component-1-class', 'app/keyup.components.ts (class v.1 - strongly typed )')(format=".")
   :marked
     <br>Strong typing reveals a serious problem with passing a DOM event into the method:
     too much awareness of template details, too little separation of concerns.
 
-    We'll address this problem in our next try at processing user keystrokes.
+    You'll address this problem in our next try at processing user keystrokes.
```

Comment:
Grep for "our", replace with "your" or "the" or whatever you deem appropriate.

(grep for "we", while you're at it)